modules/system-disaster-recovery/src/main/java/org/apache/ignite/internal/disaster/system/SystemDisasterRecoveryStorage.java

```diff
@@ -53,7 +53,10 @@ public void removeResetClusterMessage() {
         vault.remove(RESET_CLUSTER_MESSAGE_VAULT_KEY);
     }
 
-    /** Reads cluster state. */
+    /**
+     * Reads cluster state. This is used for cases when it may be needed to read it during node startup (and the usual CMG state storage
```

Comment:
Let's specify in the documentation that we read from the vault and what `null` means.